Modules/Microsoft365DSC/Examples/Resources/TeamsProtectionPolicy/2-Update.ps1

```diff
@@ -0,0 +1,27 @@
+<#
+This example is used to test new resources and showcase the usage of new resources being worked on.
+It is not meant to use as a production baseline.
+#>
+
+Configuration Example
+{
+    param(
+        [Parameter(Mandatory = $true)]
+        [PSCredential]
+        $credsCredential
+    )
+    Import-DscResource -ModuleName Microsoft365DSC
+
+    node localhost
+    {
+        TeamsProtectionPolicy "TeamsProtectionPolicy"
+        {
+            IsSingleInstance                 = "Yes"
+            AdminDisplayName                 = "Contoso Administrator"
+            HighConfidencePhishQuarantineTag = "DefaultFullAccessPolicy"
+            MalwareQuarantineTag             = "AdminOnlyAccessPolicy"
+            ZapEnabled                       = $true
+            Credential                       = $credsCredential
```

Comment:
All examples must use certificate authentication, Nik requested me to change all of them a few months ago.